scripts/exglobal_atmos_chgres_gen_control.sh

```diff
@@ -0,0 +1,98 @@
+# Script name:         exglobal_atmos_chgres_gen_control.sh
+# Script description:  Runs chgres on changing resolution of GEFS stage ic control member
+################################################################################
+# Dependent input scripts and Executables
+CHGRESEXEC="${HOMEgfs}/sorc/ufs_utils.fd/exec/chgres_cube"
+################################################################################
+# copy input files to DATA from the source directory
+cpfs "${FIXgfs}/am/global_hyblev.l${LEVS}.txt" "${DATA}/"
+cpfs "${FIXgfs}/orog/${CASE}/${CASE}_mosaic.nc" "${DATA}/"
+# uncomment and modify when the correct files are available
+# cpfs "${COMIN_ATMOS_HISTORY_MEM}/${SFC_FILE}" "${DATA}/"
+# cpfs "${COMIN_ATMOS_HISTORY_MEM}/${ATM_FILE}" "${DATA}/"
+# SFC_FILE="gdas.t18z.sfcf003.nc"
+# ATM_FILE="gdas.t18z.atmf003.ensres.nc"
+###############################################################################
+# copy orography,surface, and ancillary files to DATA from the source directory
+for i in {1..6}; do
+  cpfs "${FIXgfs}/orog/${CASE}/${CASE}_grid.tile${i}.nc" "${DATA}/"
+  cpfs "${FIXgfs}/orog/${CASE}/${CASE}.mx${OCNRES}_oro_data.tile${i}.nc" "${DATA}/"
+  cpfs "${FIXgfs}/orog/${CASE}/sfc/${CASE}.mx${OCNRES}.slope_type.tile${i}.nc" "${DATA}/"
+  cpfs "${FIXgfs}/orog/${CASE}/sfc/${CASE}.mx${OCNRES}.maximum_snow_albedo.tile${i}.nc" "${DATA}/"
+  cpfs "${FIXgfs}/orog/${CASE}/sfc/${CASE}.mx${OCNRES}.snowfree_albedo.tile${i}.nc" "${DATA}/"
+  cpfs "${FIXgfs}/orog/${CASE}/sfc/${CASE}.mx${OCNRES}.soil_type.tile${i}.nc" "${DATA}/"
+  cpfs "${FIXgfs}/orog/${CASE}/sfc/${CASE}.mx${OCNRES}.vegetation_type.tile${i}.nc" "${DATA}/"
+  cpfs "${FIXgfs}/orog/${CASE}/sfc/${CASE}.mx${OCNRES}.substrate_temperature.tile${i}.nc" "${DATA}/"
+  cpfs "${FIXgfs}/orog/${CASE}/sfc/${CASE}.mx${OCNRES}.vegetation_greenness.tile${i}.nc" "${DATA}/"
+  cpfs "${FIXgfs}/orog/${CASE}/sfc/${CASE}.mx${OCNRES}.facsf.tile${i}.nc" "${DATA}/"
+done
+################################################################################
+# add the namelist and run chgres
+cat << EOF > ./fort.41
+&config
+  mosaic_file_target_grid="./${CASE}_mosaic.nc"
+  fix_dir_target_grid="./"
+  orog_dir_target_grid="./"
+  orog_files_target_grid="${CASE}.mx${OCNRES}_oro_data.tile1.nc",\
+                         "${CASE}.mx${OCNRES}_oro_data.tile2.nc",\
+                         "${CASE}.mx${OCNRES}_oro_data.tile3.nc",\
+                         "${CASE}.mx${OCNRES}_oro_data.tile4.nc",\
+                         "${CASE}.mx${OCNRES}_oro_data.tile5.nc",\
+                         "${CASE}.mx${OCNRES}_oro_data.tile6.nc"
+  vcoord_file_target_grid="./global_hyblev.l${LEVS}.txt"
+  mosaic_file_input_grid="NULL"
+  orog_dir_input_grid="NULL"
+  orog_files_input_grid="NULL"
+  data_dir_input_grid="./"
+  atm_files_input_grid="${ATM_FILE}"
+  atm_core_files_input_grid="NULL"
+  atm_tracer_files_input_grid="NULL"
+  sfc_files_input_grid="${SFC_FILE}"
```

Comment:
```suggestion
  atm_files_input_grid="./atm_input.nc"
  sfc_files_input_grid="./sfc_input.nc"
  atm_core_files_input_grid="NULL"
  atm_tracer_files_input_grid="NULL"
```
Lets copy the `ATM_FILE` and `SFC_FILE` as `atm_input.nc` and `sfc_input.nc` so this becomes irrelevant.